projects/packages/my-jetpack/_inc/components/connection-status-card/index.jsx

```diff
@@ -137,16 +145,17 @@ const ConnectionStatusCard = props => {
 						<ConnectionListItem
 							onClick={ openManageConnectionDialog }
 							text={ __( 'Site connected.', 'jetpack-my-jetpack' ) }
-							actionText={ ! isUserConnected ? __( 'Manage', 'jetpack-my-jetpack' ) : null }
+							actionText={ isUserConnected ? __( 'Manage', 'jetpack-my-jetpack' ) : null }
 						/>
 						{ isUserConnected && (
 							<ConnectionListItem
 								onClick={ openManageConnectionDialog }
 								actionText={ __( 'Manage', 'jetpack-my-jetpack' ) }
 								text={ sprintf(
-									/* translators: placeholder is user name */
-									__( 'Connected as %s.', 'jetpack-my-jetpack' ),
-									userConnectionData.currentUser?.wpcomUser?.display_name
+									/* translators: first placeholder is user name, second is either the (Owner) string or an empty string */
+									__( 'Connected as %1$s %2$s.', 'jetpack-my-jetpack' ),
+									userConnectionData.currentUser?.wpcomUser?.display_name,
+									userConnectionData.currentUser?.isMaster ? '(Owner)' : ''
```

Comment:
```suggestion
									__( 'Connected as %1$s%2$s.', 'jetpack-my-jetpack' ),
									userConnectionData.currentUser?.wpcomUser?.display_name,
									userConnectionData.currentUser?.isMaster ? ' (Owner)' : ''
```


To avoid an empty space before the period:
![2022-12-28_14-35-35](https://user-images.githubusercontent.com/8486249/209851019-bcf15453-de26-4384-85ce-9b750f9ac3e2.png)

Reply:
The ` (Owner)` should be a call to `__` for translation as well.